src/Config/ProjectsConfiguration.php

```diff
@@ -75,7 +75,7 @@ private function addVersionsNode()
         $builder = new TreeBuilder('versions');
         $node = $builder->getRootNode();
 
-        $childrenNode = $node->children();
+        $childrenNode = $node->addDefaultsIfNotSet()->children();
```

Comment:
Also fixed a bug here, where if the package does not have versions (AKA test this package with symfony 4.4 and 5.1 for example) it gets no `version` key and it breaks on my env atleast, maybe on prod it does not, but still, it should be considered as a bug